tools/cldr/build.xml

```diff
@@ -39,9 +39,31 @@
         </condition>
         <fail unless="cldrDir"
              message="Set the CLDR_DIR environment variable (or cldrDir property) to the top level CLDR source directory (containing 'common')."/>
+        <property name="testDataDir4C" value="${basedir}/../../icu4c/source/test/testdata/cldr"/>
+        <property name="testDataDir4J" value="${basedir}/../../icu4j/main/tests/core/src/com/ibm/icu/dev/data/cldr"/>
     </target>
 
-    <!-- TODO: Add properties and targets for copying test data. -->
+    <!-- Copies CLDR test data -->
```

Comment:
Maybe say:
    
`<!-- Copies CLDR test data directories (deleting previous contents first to prevent inconsistent state) -->`

Ant uses timestamps and won't copy "old" files over "new" ones, but this is prone to all sorts of fun, so it's very important that the directory is "cleaned" first. The alternative is using the "overwrite" flag, but you can still leave old versions around with that. So what you're doing is definitely the only sensible approach, it's just that it takes a bit of thinking to appreciate that.

It also makes the comment "Add directories here..." important since you don't want an explicit list of files, because when that changes you'd potentially leave old files un-deleted.

Your directory-at-a-time approach is safe and good, and I think the only reasonable solution, so maybe change:
`<!-- The set of CLDR test data files to be copied into ICU. -->`
to:
`<!-- CLDR test data directories to be copied into ICU (do not list individual files here). -->`

Reply:
Done. I've changed the README too, to emphasize directories / de-emphasize files.

However are directories that much better? If someone removes a directory from the list, same problem would occur, it wouldn't get deleted? So I'm emphasizing directories but not banning files... does this seem reasonable?

(Not squashing yet, for ease of reviewing the latest changes.)